lib/plugins/exchange/kraken.js

```diff
@@ -9,6 +9,7 @@ const CRYPTO = [BTC, ETH, LTC, DASH, ZEC, BCH]
 const FIAT = ['USD', 'EUR']
 const AMOUNT_PRECISION = 6
 const REQUIRED_CONFIG_FIELDS = ['apiKey', 'privateKey']
+const CUSTOM_KEY = 'userref'

```

Comment:
Why is this called ``CUSTOM_KEY``? Shouldn't it be called ``USER_REF``?

Reply:
The goal is to set some sort of pattern among exchange files, if in the future Itbit, e.g, adds a similar field, we could have the same CUSTOM_KEY with whatever name, and we wouldn't need to change anything on the ccxt file.

Reply:
After discussing on chat, we decided to go with ``USER_REF`` in the end.